compiler/packages/babel-plugin-react-compiler/package.json

```diff
@@ -50,6 +50,7 @@
     "pretty-format": "^24",
     "react": "0.0.0-experimental-4beb1fd8-20241118",
     "react-dom": "0.0.0-experimental-4beb1fd8-20241118",
+    "react-compiler-runtime": "0.0.1",
```

Comment:
We might also need some way of ensuring `react-compiler-runtime` is bundled (if any bundlers do pre-babel treeshaking).